### DISCUSSION TITLE
Multiple windows for plotting

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#964] [Q&A]
Hi, Is there any way to open multiple window for monitoring the plotter ? I want to monitor more config of the plotter simultaneously. But now if I watch the one config the other will not any record. How can I do if I want to monitoring the rang, ranging, stabilizer, TWR (like picture config that I create by myself )this information simultaneously. 
 
![A544AD64-DA36-4805-920C-1BBABB41065E](https://github.com/bitcraze/discussions/assets/118436188/9f3fd13b-a4fb-4536-94cd-5279c3b884f8)

**Comment by knmcguire** [comment]
That is a good question! Technically it should be possible with the python library but this is not an existing feature in the cfclient itself ... 

Crazyswarm2 is able to do this with the ROS2 framework, so I know this is indeed possible. I've made an enhancement ticket on the cfclient github repo such that we or somebody else can contribute this.

https://github.com/bitcraze/crazyflie-clients-python/issues/681
